models/articles-model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (req, res) => {
+    return db.query(
+        `SELECT * from articles 
+        WHERE article_id = $1`, [req])
+    .then(({ rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "Not found."})
+        }
+        return rows;
```

Comment:
Ideally we should only be sending back the first element of the array here which is the response object. We only want to send an array if there is multiple responses.